.docker/Dockerfile-build

```diff
@@ -1,4 +1,5 @@
-FROM golang:1.16.10-alpine3.14 AS builder
+FROM golang:1.17-alpine3.15 AS base
+# syntax = docker/dockerfile:1-experimental
```

Comment:
This should be the first line in the file.

Reply:
Personally I do not like to use it and instead rely on Docker 20+ already supporting those additional features. The reason why I do not like it is because if you provide it it makes Docker always pull the `docker/dockerfile` image first, before doing anything else, which delays the build process.

Reply:
Looks like this has been merged with the "comment" in the wrong line. This way it does nothing and is very missleading.